tests/cephadm/test_cephadm_upgrade.py

```diff
@@ -3,10 +3,88 @@
 
 """
 import logging
+from concurrent.futures import ThreadPoolExecutor
+from time import time
 
 from ceph.ceph_admin.orch import Orch
 
-LOG = logging.getLogger()
+LOG = logging.getLogger(__name__)
+
+
+class RadosBench:
+    def __init__(self, client, pool=""):
+        """
+        Initialize RadosBench
+        """
+        self.client = client
+        self.pool = pool or self.create_pool()
```

Comment:
This looks more like an utility... hence, we could attempt not to initialize with client of pool... The attribute of this could be 
- start_time
- end_time
- interval
- max_time

Reply:
we can discuss more about this.